src/index.html

```diff
@@ -11,8 +14,53 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute in the link tag should point to a CSS file, not an SCSS file. Ensure that the SCSS is compiled to CSS and update the path accordingly.